components/03-sections/01-accordion/accordion.hbs

```diff
@@ -1,15 +1,22 @@
 <section class="select-publication-sec scholarship-type-designs">
     <div class="container">
-        <div id="accordion">
-            <div class="card">
-                <div class="card-header"> <a class="card-link" data-bs-toggle="collapse"
-                        href="#collapse-01">{{ heading }}</a>
-                </div>
-                <div id="collapse-01" class="collapse" data-bs-parent="#accordion">
+        <div class="accordion" id="accordion">
+            <div class="accordion-item">
+                <h5 class="accordion-header" id="heading-01">
+                    <button type="button" class="accordion-button collapsed" data-bs-toggle="collapse"
+                        data-bs-target="#collapse-01">
+                        {{heading}}
+                    </button>
+                </h5>
+                <div id="collapse-01" class="accordion-collapse collapse" data-bs-parent="#accordion">
                     <div class="card-body">
                         <div class="accordion-body-content">
-                            <h5>{{ title }}</h5>
-                            <p>{{ text }}</p>
+                            <h5>
+                                {{title}}
+                            </h5>
+                            <p>
+                                {{text}}
+                            </p>
                             <ul class="ul-list">
                                 <li class="list-item">Use the HTML <ol> element to define an ordered list</li>
```

Comment:
Please add backticks (``) around `<ol>` element, so it doesn't cause a code break.